app/views/declaration/additional_information.scala.html

```diff
@@ -49,19 +49,19 @@
                 <table id="additional_information">
                     <thead>
                         <tr>
-                            <th class="code-header" colspan="1">@messages("supplementary.additionalInformation.table.headers.code")</th>
-                            <th class="description-header" colspan="1">@messages("supplementary.additionalInformation.table.headers.description")</th>
-                            <th colspan="1"></th>
+                            <th class="code-header" scope="col">@messages("supplementary.additionalInformation.table.headers.code")</th>
+                            <th class="description-header" scope="col">@messages("supplementary.additionalInformation.table.headers.description")</th>
+                            <td></td>
```

Comment:
If it's in `thead` is should be an empty `th`?

Reply:
It can be `TD`.
It was an empty `TH`, which caused a warning/alert. We have two options:
1. remove that and add `colspan="2"`
2. add `td` which can be empty

Two shifts the button to the right, which we prefer.